README.md

```diff
@@ -1,2 +1,14 @@
 # CodeCoverage
-C++ CodeCoverage
+The CodeCoverage utility is used for generating code coverage reports for C++ repositories. Currently the following LogRhythm repositories can get code coverage measurements through this utility: 
+* [StopWatch](https://github.com/LogRhythm/StopWatch)
+* [Queuenado](https://github.com/LogRhythm/Queuenado)
+* [FileIO](https://github.com/LogRhythm/FileIO)
+* [DeathKnell](https://github.com/LogRhythm/DeathKnell)
+
+## Usage example
+```
+git clone https://github.com/LogRhythm/FileIO
+cd FileIO
+/usr/local/probe/bin/CodeCoverage.py
```

Comment:
This is not working for me. I have unit tests failing when running as a normal user (non-root).
```
[ RUN      ] TestFileIO.TestReadAsciiFileContentAsRoot
/home/jgress/wk/LROSS/FileIO/test/ToolsTestFileIO.cpp:630: Failure
Value of: 0
Expected: previousUID
Which is: 1000
[  FAILED  ] TestFileIO.TestReadAsciiFileContentAsRoot (5 ms)
```
If I run as root, that unit test passes, but in either case, the code coverage report is not generated:
```
Exiting, log location: /tmp/UnitTest1000.log
profiling:/home/jgress/wk/LROSS/FileIO/build/CMakeFiles/gtest_170_lib.dir/3rdparty/gtest-1.7.0/src/gtest-all.cc.gcda:Cannot open
profiling:/home/jgress/wk/LROSS/FileIO/build/CMakeFiles/UnitTestRunner.dir/test/ToolsTestFileSystemWalker.cpp.gcda:Cannot open
profiling:/home/jgress/wk/LROSS/FileIO/build/CMakeFiles/UnitTestRunner.dir/test/ToolsTestFileIO.cpp.gcda:Cannot open
profiling:/home/jgress/wk/LROSS/FileIO/build/CMakeFiles/UnitTestRunner.dir/3rdparty/test_main.cpp.gcda:Cannot open
profiling:/home/jgress/wk/LROSS/FileIO/build/CMakeFiles/FileIO.dir/src/FileSystemWalker.cpp.gcda:Cannot open
profiling:/home/jgress/wk/LROSS/FileIO/build/CMakeFiles/FileIO.dir/src/FileIO.cpp.gcda:Cannot open
profiling:/home/jgress/wk/LROSS/FileIO/build/CMakeFiles/FileIO.dir/src/DirectoryReader.cpp.gcda:Cannot open
ERROR: UnitTestRunner process failed!
Copy coverage files into coverage directory return code: 0
/bin/sh: /usr/bin/gcovr: No such file or directory
ERROR: Gcovr process failed!
```